pkg/littlefs/fs/littlefs_fs.c

```diff
@@ -146,7 +146,13 @@ static int prepare(littlefs_desc_t *fs)
     if (!fs->config.read_size) {
         fs->config.read_size = fs->dev->page_size;
     }
-    fs->config.lookahead = LITTLEFS_LOOKAHEAD_SIZE;
+    if (!fs->config.cache_size) {
+        fs->config.cache_size = fs->dev->page_size * LITTLEFS_CACHE_PAGES;
+    }
+    if (!fs->config.block_cycles) {
+        fs->config.block_cycles = LITTLEFS_BLOCK_CYCLES;
+    }
+    fs->config.lookahead_size = LITTLEFS_LOOKAHEAD_SIZE;
```

Comment:
[Previously](https://github.com/ARMmbed/littlefs/blob/v1.7.2/lfs.h#L155-L159), `config::lookahead` was defined as a number of bits. This was [changed](https://github.com/ARMmbed/littlefs/blob/v2.1.3/lfs.h#L209-L213) with the move to `config::lookahead_size` to bytes. This should be reflected in the allocated buffer [here](https://github.com/RIOT-OS/RIOT/blob/e8c78ad2ae88868770f21a00a92e13d17835365e/sys/include/fs/littlefs_fs.h#L100), the divide by 8 should be removed, or it should be added here.

With the current state of this PR, the memset [here](https://github.com/ARMmbed/littlefs/blob/v2.1.3/lfs.c#L3518) will overwrite large parts of the memory, exceeding the buffer limits.